### PR TITLE
add dev.boxi.Boxi

### PR DESCRIPTION
**Pull request by allisonkarlitskaya**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
  - `--talk-name=org.freedesktop.Flatpak` is the big one here: this is a terminal emulator which starts sessions on the host (via `flatpak-spawn --host`)
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by JakobDev** [comment]
bot, build dev.boxi.Boxi

**Comment by JakobDev** [comment]
I suggest that you use x-checker-data in your manifest

File: dev.boxi.Boxi.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env -S flatpak-builder --user --install --force-clean tmp
+
+app-id: dev.boxi.Boxi
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+
+modules:
```
Comment (hfiguiere) [inline dev.boxi.Boxi.yml:8]: I know it's too late, but usualy modules are last in manifest.

File: dev.boxi.Boxi.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        sha256: e454fdbf68c7036e1c7435ec7479383f9d9a1650ca5b304feb184eba1efcdcef
+    buildsystem: simple
+    build-commands:
+      - pip3 install --no-build-isolation --prefix=/app flit_core-3.7.1-py3-none-any.whl
+
+  - name: boxi
+    sources:
+      - type: git
+        url: https://github.com/allisonkarlitskaya/boxi
+        tag: 3
```
Comment (hfiguiere) [inline dev.boxi.Boxi.yml:30]: I bet that yaml is throwing a warning. It should be `tag: '3'` since it expect a string.

**Pull request closed without merging**